### ISSUE TITLE
my psvita in 3.74 crached when i install h-encore

### ISSUE THREAD

**Issue by Kinsley12off** [soarqin/finalhe#126]
i cant exit h-encore and i need reboot my psvita to use it again

**Comment by IlNevioIl** [comment]
Once i click the exit option inside h encore, it just doesnt exit and freezes.

**Comment by Kinsley12off** [comment]
yeah, this is the same for me i tried a few times and the same happens and if i wait a few moments and i press the ps button i can go in the psv start menu but i can't open apps and shutdown my psvita

**Comment by Knofbath** [comment]
Well, pending TheFlow releasing another version of h-encore, this is best there is. FinalHE is just the delivery method. Once you get a working activation, downgrade to 3.60 or 3.60 immediately.

Power off. Retry until it works. Maybe delete the app, redownload, and reinstall. Or maybe stop trying to "exit" the program and just close it using the PS button.
